source/tutorials/kubernetes.rst

```diff
@@ -255,15 +497,23 @@ For example, to customize CRI-O configuration in your system, run the following
 Learn more about :ref:`stateless` in |CL|.
 
 Proxy configuration (optional)
```

Comment:
It seems like a real burden to try to explain proxies in here. I don't think this will cover every scenario.  I know in our lab setups with multiple nodes you have to add each node's ip to the `hosts` file to deal with proxies.

Reply:
I agree, this was existing content so I just did my best to format it nicely.